src/sagemaker/transformer.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, model_name, instance_count, instance_type, strategy=None, ass
         self.sagemaker_session = sagemaker_session or Session()
 
     def transform(self, data, data_type='S3Prefix', content_type=None, compression_type=None, split_type=None,
-                  job_name=None):
+                  wait=True, logs=True, job_name=None):
```

Comment:
generally we try to add new args to the end of the list in case anyone is calling the method by relying on the order of the arguments (e.g. Python lets you call `transform(data, 'S3Prefix', 'text/csv')` without specifying the kwarg name)